src/status_im/translations/en.cljs

```diff
@@ -533,7 +533,7 @@
    :transactions-sign-later              "Sign later"
    :transactions-delete                  "Delete transaction"
    :transactions-delete-content          "Transaction will be removed from 'Unsigned' list"
-   :transactions-history                 "History"
+   :transactions-history                 "Transaction history"
```

Comment:
`Transaction` ?

Reply:
Yes. See the title in second screenshot. Previously there was just Transaction in title and History in one of the tabs (the other tab was Unsigned - now removed).

Since I removed the tab that says History, that word needs to go to the title.

Reply:
I know we want to keep translation entries immutable, but this key for string "History" was already named like the new string. Have a better idea?

Reply:
no, just key name confused me